### ISSUE TITLE
Add item_id to license updater

### ISSUE THREAD

**Issue by brashrebel** [realbigplugins/psp-edd#5]
Simpler and better than the title matching.

**Comment by d4mation** [comment]
This is supported in RBP Support:

https://github.com/realbigplugins/rbp-support/blob/7aa129c5d6202ac227a04811d965efd06bb6efbf/rbp-support.php#L645-L651

**Comment by brashrebel** [comment]
@d4mation so if that is included here in psp_edd, can we close this issue?

**Comment by brashrebel** [comment]
Also, if the licensing is built into RBP_Support, than I don't need to setup licensing in the plugin I am working on right now? PSP Scheduled Notifications still needs RBP Support added and I have never done that. You mind guiding me through https://github.com/realbigplugins/psp-scheduled-notifications/issues/1 ?

**Comment by d4mation** [comment]
Nope, it is supported, but I did not set it up as Download Name is working. 

Getting it to use Download ID is simply running a Filter specific to your RBP Support instance though.

**Comment by brashrebel** [comment]
I would recommend converting that to use ID instead of download name globally. Not urgent but preferred.

**Comment by d4mation** [comment]
Yeah, RBP Support does both Licensing and Support Forms. It was easier to just bundle the both of them as the Support Form needed to know whether or not the License is Active and having it also control Licensing ensures that the data is formatted in a predictable way. The README for the RBP Support Repo goes over how to set it up, but if you have questions let me know :+1:

Why is Download ID "better", exactly? In most example implementations of EDD_License/EDD_SL_Plugin_Updater I've seen they all use Download Name. It seems to be the normal implementation and that Download ID is optional.
